### ISSUE TITLE
on conda-forge

### ISSUE THREAD

**Issue by ReimarBauer** [althonos/fs.sshfs#9]
Hi
the prior version 0.6.1 made it into a conda-forge feedstock :+1:  \o/ 

https://anaconda.org/conda-forge/fs.sshfs

I have not added all the strong version dependencies similiar as you did in setuptools yet, which gives currently some issues I have to fix :) Are those strong settings still required? fs has a newer version too.

https://github.com/conda-forge/fs.sshfs-feedstock/issues

cheers
Reimar

**Comment by althonos** [comment]
@ReimarBauer : glab this made it, I'll probably add a badge about this. Thanks.

I know about the new fs version, most of the changes in `v2.0.15` are from a PR of mine 😉 

For the depencies, I answered in the issue tracker of the feedstock. I manually tested them, and made sure they are the lowest required version:

* `six == 1.10` is the minimal version required by `fs`
* `paramiko < 2` is absolutely not compatible with `paramiko == 2`
* `fs < 2.0.7` doesn't implement the extension mechanism, so it makes `fs.sshfs` useless
* `configparser` does not need to be pinned
